imap_processing/cdf/config/imap_mag_global_cdf_attrs.yaml

```diff
@@ -0,0 +1,46 @@
+Descriptor: MAG>Magnetometer
+TEXT: >
+  The IMAP magnetometer (MAG consists of a pair of identical magnetometers
+  which each measure the magnetic field in three directions in the vicinity of
+  the spacecraft.
+  MAG will contribute to our understanding of the acceleration and transport
+  of charged particles in the heliosphere.
+  MAG design and assembly is led by Imperial College, London. See
+  https://imap.princeton.edu/instruments/mag for more details.
+
+imap_mag_l1a_norm-raw:
```

Comment:
Now that I am working on replicating this yaml for Hi, I have a comment:
It is not apparent in the global attrs yaml how the dependency works like it is in the variable attrs yaml. I really like being able to see that dependency directly in the yaml without having to snoop in the code to figure out how things like `Descriptor` and `TEXT` get propagated into product specific global attrs.

Reply:
Maybe we could rework the global attrs to work the same way, with the default and then the specifics? Then when retrieving the attribute, you would always specify the logical_source_id.

Reply:
I put a change in to hopefully mitigate this? @subagonsouth I'd appreciate your thoughts